### ISSUE TITLE
Power and shield values cap, long integer?

### ISSUE THREAD

**Issue by Whackjob** [tsunamayo/Starship-EVO#2814]
### Short Description of the issue:
My shields and generator appear to cap at 65,536,000.
...

### Steps to reproduce the issue:
Build fairly large.
...

### Environment
- Starship EVO version: 20w43b
- Graphics Card: Quadro P5000

### Additional information:

[Screenshot of ship stats ](https://i.imgur.com/J1ZTieY.png) 

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

**Comment by tsunamayo** [comment]
@Whackjob can you send me your blueprint, it will be faster to reproduce on my side! cheers

**Comment by Whackjob** [comment]
Absolutely.  I've an update as well... I don't think it's actually capped at the long integer level.  I've gone past it a few times.  It now seems like it just reports at that level some times.
